autoware_planning_msgs/msg/Segment.msg

```diff
@@ -0,0 +1,6 @@
+# Segments are similar to lanelets.
+
+uint16 id
+Linestring[2] linestrings
+int32[] successor_lanelet_id
+int32[] neighboring_lanelet_id
```

Comment:
we should avoid using the term "lanelet" (also remove from the comment on line 1 and in `RoadSegments` (line 2).
```suggestion
int32[] successor_segment_id
int32[] neighboring_segment_id
```

Reply:
Done!